apps/dapp-console/app/settings/components/ClaimRebateDialog.tsx

```diff
@@ -28,6 +28,13 @@ export type ClaimRebateDialogProps = {
   onRebateClaimed: (contract: Contract) => void
 }
 
+const errorMessages: Record<string, string> = {
```

Comment:
nit: if you update this type to  `Partial<Record<DAPP_CONSOLE_ERROR_CODE, string>>`, then you will be able to make sure that the keys in this match the error codes